.github/workflows/mypy.yml

```diff
@@ -25,4 +25,5 @@ jobs:
         with:
           checkName: 'mypy'   # NOTE: this needs to be the same as the job name
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        continue-on-error: true # https://github.com/Draegerwerk/sdc11073/issues/156
```

Comment:
remove this, as it was already done in https://github.com/Draegerwerk/sdc11073/pull/216 when this PR was opened

Reply:
done